src/client/index.tsx

```diff
@@ -44,7 +44,7 @@ function renderOutput(outputItem: OutputItem, element: HTMLElement, ctx: Rendere
         console.log(`Rendering mimeType ${mimeString}`, output);
 
         ReactDOM.render(
-            React.createElement(CellOutput, { mimeType: mimeString, output, ctx, outputId: outputItem.id }, null),
+            React.createElement(CellOutput, { mimeType: mimeString, output, ctx, outputId: (outputItem.metadata as any)?.id || outputItem.id }, null),
```

Comment:
Some questions:

- Is it ok to use `any` here? Or should I make a new type?
- Is it ok to default to `outputItem.id`? I know the idea is to stop using `outputItem.id`, but given that the code that sets `metadata.id` is not yet merged, this seemed safe to me.